### DISCUSSION TITLE
canAccessPanel method is not call automatically

### DISCUSSION THREAD

**Discussion by drshahidmca** [filamentphp/filament#15095] [Help]
### Package

Panel builder

### Package Version

 v3.2.130

### How can we help you?

canAccessPanel method is not call automatically for Authorizing access to the panel.Iamm usinf filament latest version

**Comment by leandrocfe** [comment]
Did you check [this section](https://filamentphp.com/docs/3.x/panels/users#authorizing-access-to-the-panel) ?
Share the code please
